tests/cypress/e2e/features/ground_truth_jobs.js

```diff
@@ -323,23 +353,52 @@ context('Ground truth jobs', () => {
             cy.get('.cvat-objects-sidebar-show-ground-truth').filter(':visible').click();
 
             cy.goCheckFrameNumber(groundTruthFrames[0]);
-            checkConflicts('warning', 1);
+            checkConflicts('error', 2);
+            checkHighlight(1);
 
             cy.goCheckFrameNumber(groundTruthFrames[1]);
-            checkConflicts();
+            checkConflicts('warning', 1);
 
             cy.goCheckFrameNumber(groundTruthFrames[2]);
+            checkConflicts();
+
+            cy.goCheckFrameNumber(groundTruthFrames[3]);
             checkConflicts('error', 2);
+            checkHighlight(1);
+        });
+
+        it('Conflicts with annotation filter enabled', () => {
+            cy.addFiltersRule(0);
+            cy.setFilter({
+                groupIndex: 0,
+                ruleIndex: 0,
+                field: 'ObjectID',
+                operator: '>',
+                value: '5',
+                submit: true,
+            });
+
+            cy.goCheckFrameNumber(groundTruthFrames[0]);
+            checkConflicts('', 0, false);
+
+            cy.goCheckFrameNumber(groundTruthFrames[1]);
+            checkConflicts('', 0, false);
+
+            cy.goCheckFrameNumber(groundTruthFrames[2]);
+            checkConflicts('', 0, false);
+
+            cy.goCheckFrameNumber(groundTruthFrames[3]);
+            checkConflicts('error', 1, false);
         });
 
         it('Frames with conflicts navigation', () => {
             cy.goCheckFrameNumber(0);
```

Comment:
Do we need to have `groundTruthFrames[0]` instead of `0` here?

Reply:
Previously we needed that because checking navigation was for gt frames from non-gt frame - 0. But now 0 is gt frame so it doesnt really matter what we use here. Added `groundTruthFrames[0]` instead